src/main/scala/com/fulcrumgenomics/vcf/filtration/EndRepairArtifactLikelihoodFilter.scala

```diff
@@ -131,39 +136,72 @@ class EndRepairArtifactLikelihoodFilter(val distance: Int = 2,
     * artifact prior of 1 - mutation prior.  This has the effect of creating priors that are heavily
     * skewed towards the artifact at very low MAFs, and towards mutations as the MAF increases towards 0.5.
     */
-  private[filtration] def priors(pileup: Pileup[PileupEntry], gt: Genotype, maf: Double) : (Double, Double) = {
+  private[filtration] def priors(pileup: Pileup[PileupEntry], maf: Double) : (Double, Double) = {
     require(maf > 0 && maf <= 1, "maf must be greater than 0 and no more than 1.")
     val priorMutation = min(pow(maf * 2, 2), 0.9999)
     val priorArtifact = 1 - priorMutation
     (priorMutation, priorArtifact)
   }
 
-  /** Exponentiates a log probability that should be in the range 0-1 and caps it there to account for numerical imprecision. */
-  private def expProb(lnP: LnProb): Double = {
-    val x = exp(lnP)
-    if (x > 1) 1 else if (x < 0) 0 else x
-  }
-
   /**
     * Returns true if a base in the pileup (either ref or alt) is congruent with the artifact.
     * I.e. the base is within the first or last few bases of the template _and_ the base as read
     * makes sense given the position.
+    *
+    * Given a ref allele and alt allele (alt will always be A or T when invoking this method) we infer
+    * how the alleles are paired.  E.g. if we are give ref=C alt=A then we pair (C,A) and (G,T)
+    * together and think about C and A in similar ways, and G and T in similar ways.
+    *
+    * Next, it's important to know if we are looking at an observation at the beginning or end of
+    * the read (e.g. if the read goes all the way through the template), as this informs how the
+    * artifact will present.  See the two examples below for a C>A artifact:
+    *
+    * Example 1: congruent with the artifact
+    *     Ref(+)     CCCCCCCCCCCCCCCCCC
+    *     R1(+)   5'-CCCCCCCCCCCCCCCCCa-3'
+    *     R2(-)   3'-GGGGGGGGGGGGGGGGGt-5' (in BAM would also be CCCCCCCCCCCCCCCCCa)
+    *
+    * Example 2: incongruent with the artifact
+    *     Ref(+)     CCCCCCCCCCCCCCCCCC
+    *     R1(+)   5'-aCCCCCCCCCCCCCCCCC-3'
+    *     R2(-)   3'-tGGGGGGGGGGGGGGGGG-5' (in BAM would also be aCCCCCCCCCCCCCCCCC)
+    *
+    * In Example 1 we see a C>A substitution at the end of the template, which R1 observes at a C>A substitution
+    * _at the end of the read_ and R2 observes as a G>T substitution _at the beginning of the read_.  This is
+    * congruent with the model of over-digestion of the 3' end and addition of one or more A nucleotides.
+    *
+    * In Example 2 we see a C>A substitution at the beginning of the template, which R1 observes at a C>A substitution
+    * _at the beginning of the read_ and R2 observes as a G>T substitution _at the end of the read_.  This is
+    * NOT congruent with the model of over-digestion of the 3' end and addition of one or more A nucleotides.
```

Comment:
An example 3 would drive this home for a `ref=C alt=T`, but your call.  Thank-you for writing this up!

```
Ref(+)     CCCCCCCCCCCCCCCCCC
R1(+)   5'-tCCCCCCCCCCCCCCCCC-3'
R2(-)   3'-aGGGGGGGGGGGGGGGGG-5'
```